### ISSUE TITLE
`setting-path` option

### ISSUE THREAD

**Issue by 10000-ki** [graalvm/setup-graalvm#25]
Is there an option such as setting-path

like [setup-java](https://oss.navercorp.com/actions/setup-java)

https://oss.navercorp.com/actions/setup-java/blob/8d222861066e40836af2181b3a2ddd46f05b2dd8/docs/advanced-usage.md#yaml-example

> NOTE: The settings.xml file is created in the Actions $HOME/.m2 directory. If you have an existing settings.xml file at that location, it will be overwritten. See [below](https://oss.navercorp.com/actions/setup-java/blob/8d222861066e40836af2181b3a2ddd46f05b2dd8/docs/advanced-usage.md#apache-maven-with-a-settings-path) for using the settings-path to change your settings.xml file location.
If you don't want to overwrite the settings.xml file, you can set overwrite-settings: false

**Comment by fniephaus** [comment]
At the moment, there isn't. I'm not sure I understand what this option is for. Why would you have an existing `settings.xml` file at the default location?

**Comment by 10000-ki** [comment]
When using an Actions self-hosted runner with multiple shared runners the default $HOME directory can be shared by a number runners at the same time which could overwrite existing settings file.

Is there any problem like above in graalvm setup action?

**Comment by fniephaus** [comment]
Yes, I believe this extension has the describe problem. We'll look into a fix for this.

**Comment by 10000-ki** [comment]
> Yes, I believe this extension has the describe problem. We'll look into a fix for this.

thx
